samples/sample/src/commonMain/kotlin/com/careem/mockingbird/sample/PippoSample.kt

```diff
@@ -24,5 +24,8 @@ interface PippoSample {
     fun sayHiWith(param: String)
     fun sayHiWith(param: String, someOtherParam: Boolean)
     fun sayHiWithCommonParam(param: String, intParam: Int)
+    fun sayHiWith(param: String, lambda: (Int) -> Int)
```

Comment:
could you add other couple of cases like a function with 0 arguments and a function with 2 arguments for example and a function that returns `Unit` please?